### ISSUE TITLE
Browser Intelligent Automation

### ISSUE THREAD

**Issue by wkjw** [adamlui/ai-web-extensions#112]
Is it possible to automatically identify the content in the browser page, then submit it to GPT so that it can automatically get reasonable operations, and then expand the function of automatically operating the browser content based on the reply content to achieve intelligent automatic operation of the web page?

**Comment by adamlui** [comment]
Can you give an example?

**Comment by wkjw** [comment]
Of course, for example, if I open a webpage with many logical questions in it, I hope to have a plug-in that can read all the questions on the page, then identify the best answer, perform automated operations, and then submit

**Comment by adamlui** [comment]
Can you give a detailed example, with specific questions that appear, then what query is sent to GPT
